services/Facebook.json

```diff
@@ -23,6 +23,47 @@
         "removeHelpButtons",
         "cleanUrls"
       ]
+    },
+    "devPaymentsTos": {
+      "location": "https://developers.facebook.com/policy/credits",
+      "contentSelector": "._4bl9",
+      "filters": [
+        "removeHelpButtons",
+        "cleanUrls"
+      ]
+    },
+    "communityPaymentsTos": {
+      "location": "https://www.facebook.com/payments_terms",
```

Comment:
In order to avoid inflating document types, I believe this would more accurately be represented by describing a `tos` document for a separate service which could be called “Facebook Payments”.